crypto/evp/evp_enc.c

```diff
@@ -986,8 +986,9 @@ int EVP_CIPHER_CTX_set_key_length(EVP_CIPHER_CTX *c, int keylen)
 {
     int ok;
     OSSL_PARAM params[2] = { OSSL_PARAM_END, OSSL_PARAM_END };
+    size_t len = keylen;
```

Comment:
You're not casting here, but you are everywhere else.  Is that an omission here, or is casting int to size_t not actually necessary?

Reply:
This produces a warning on windows. I guess this is everywhere though..

Reply:
removed the casts to make consistent